src/components/views/ProductPage/ProductPage.js

```diff
@@ -1,10 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './ProductPage.module.scss';
+import Tabs from '../../common/Tabs/Tabs';
+import Description from '../../common/TabsDescription/Description';
+import Reviews from '../../common/TabsReviews/Reviews';
+import CustomTab from '../../common/TabsCustomTab/CustomTab';
+import Specifications from '../../common/TabsSpecifications/Specifications';
+
+const config = [
+  {
+    title: 'Description',
+    component: <Description />,
+  },
+  {
+    title: 'Reviews (0)',
+    component: <Reviews />,
+  },
+  {
+    title: 'Specifications',
+    component: <Specifications />,
+  },
+  {
+    title: 'Custom Tab',
+    component: <CustomTab />,
+  },
+];
 
 class ProductPage extends React.Component {
+  state = {
+    activeTabCategory: 'Reviews (0)',
+  };
+
+  handleChangeActiveCategory = title => {
```

Comment:
Ja bym to trzymał wewnątrz komponentu `Tabs`. teraz za każdym razem musisz robić state na trzymanie info o atywnej karcie oraz pisać funkcje na zmiane tej karty. W większości przypadków będzie to CTRL+C CTRL+V. Logika zmiany karty, czyli zapisanie aktywnej i obsługa kliknięcia w innej powinna być zapisana raczej w komponencie `Tab`.

Co innego gdybyś tworzył komponent który np. bedzie jakimś `inputem` w tedy miało by sens przekazywać tam funkcję np. `onChange`, ale Taby raczej zawsze po prostu się przełączą

Reply:
Oki powinno być lepiej